t/cobrand/sutton.t

```diff
@@ -378,14 +379,26 @@ EOF
         is $report->state, 'fixed - council', 'A state change';
         my $update = FixMyStreet::DB->resultset("Comment")->order_by('-id')->first;
         is $update->photo, '34c2a90ba9eb225b87ca1bac05fddd0e08ac865f.jpeg';
-
+        $report->push_extra_fields( # Add extra fields expected on a Bulky waste report
```

Comment:
I would move this line to 324 when the report is made a bulky collection one, so set up/update is all in the same place.